### PR TITLE
API model and Rest basics

### PR DESCRIPTION
**Pull request by landreeaa**
- Added information about the API Model 
- Added information about Rest basics

### CODE REVIEW

File: ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+<div style="text-align: center;">
+    <img width="25%" src="images/api-anim.gif">
+</div>
+
+
+***
+
+## Contents
+
+1. [Brief Overview of API Testing and Importance of Automated API Testing](#introduction-to-api-and-importance)
```
Comment (StazMD) [inline ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md:15]: Please check if links of your Table of Contents are working properly

File: ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+Let's take the scenario where you use a travel booking app to look for the best hotel offers in San Francisco. To
+receive information about hotel rates from different hotel booking websites, this app may use an API. You may compare
+prices and easily reserve the best offer thanks to the API's smooth data processing and return to the app.
+***
+<h3 style="color:AliceBlue"><strong>Why is API Testing Essential?</strong></h3>
+
+1. <p style="color:DodgerBlue">Improved Functionality </p>
+   <div style="text-align:center">
+    <img src="images/improved-functionality.png" width="15%" alt="Improved Functionality">
```
Comment (StazMD) [inline ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md:45]: Do you think that images under these points are necessary? From my point of view, they make reading slightly difficult.

**Review by endalexei — APPROVED** [review]

File: ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md (hunk 2 above)
Comment (endalexei) [reply ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md:45]: In my opinion this removes the broodiness from the reading of plain text and add some colors that make the presentation lively. I would keep them.

Comment (StazMD) [reply ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md:45]: Then let's keep them. Closing this one.

File: ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md (hunk 1 above)
Comment (landreeaa) [reply ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md]: Done
Thanks

**Review by endalexei — APPROVED** [review]

File: JT/15_how_to_identify_xpaths/How_To_Identify_XPaths_README.md [hunk 3]
```diff
@@ -7,16 +7,16 @@
 
 ## Contents
 
-1. [Introduction to XPath:Navigating the Web Landscape](#introduction-to-xpath)
-2. [Tools for XPath Identification and Testing](#tools-for-xpath)
-3. [Common XPath Patterns: A Cheat Sheet for Element Identification](#common-xpath-patterns)
-4. [XPath Challenges and Solutions: Troubleshooting in Automation](#xpath-chanllenges)
-5. [Future Trends: Evolving Role of XPath in Automation Technologies](#future-trends)
+1. [Introduction to XPath: Navigating the Web Landscape](#introduction-to-xpath-navigating-the-web-landscape)
+2. [Tools for XPath Identification and Testing](#tools-for-xpath-identification-and-testing)
```
Comment (StazMD) [inline JT/15_how_to_identify_xpaths/How_To_Identify_XPaths_README.md:11]: Please check if this link refers to correct topic title.

Comment (landreeaa) [reply JT/15_how_to_identify_xpaths/How_To_Identify_XPaths_README.md:11]: Done
Thanks

**Review by StazMD — APPROVED** [review]

**Review by bodacd — APPROVED** [review]

**Review by tascaV — APPROVED** [review]
Hat off. Bravo !

**Pull request merged**